.lando/varnish.vcl

```diff
@@ -0,0 +1,11 @@
+# @see: https://github.com/wunderio/charts/blob/master/drupal/templates/varnish-configmap-vcl.yaml
+# The routine when we deliver the HTTP request to the user
+# Last chance to modify headers that are sent to the client
+sub vcl_deliver {
+  # Called before a cached object is delivered to the client.
+
+  if (obj.hits > 0) { # Add debug header to see if it's a HIT/MISS and the number of hits, disable when not needed
+    set resp.http.X-W-Cache = "HIT";
+  } else {
+    set resp.http.X-W-Cache = "MISS";
+  }
```

Comment:
a) could we add `set resp.http.X-W-Cache-Hits = obj.hits;` for basic debugging too, used in [varnish-configmap-vcl.yaml](https://github.com/wunderio/charts/blob/9528571399354091741263577db92b313afb484e/drupal/templates/varnish-configmap-vcl.yaml#L497) as well.
b) should this have a closing bracket?

Reply:
Thanks for noticing the missing bracket ;)